activejob/lib/active_job/test_helper.rb

```diff
@@ -186,10 +189,11 @@ def assert_no_enqueued_jobs(only: nil, except: nil, queue: nil, &block)
     #     end
     #     assert_performed_jobs 1
     #
-    #     perform_enqueued_jobs do
-    #       HelloJob.perform_later('yves')
-    #       assert_performed_jobs 2
-    #     end
+    #     HelloJob.perform_later('yves')
+    #
+    #     perform_enqueued_jobs
+    #
+    #     assert_performed_jobs 2
```

Comment:
We're showing two API calling styles here. Let's provide some guidance about when you'd use one vs. the other.

Reply:
Hm. Then we should provide guidance about the one vs the other to every helper/assertion.
I believe that this difference is explicit from the current explanations like
`"If a block is passed, that block performs all of the jobs that were enqueued throughout the duration of the block and asserts that the job has been performed with the given arguments in the block."` and examples.
Let's go with as it is for now?

Reply:
Agreed 👍